scripts/do-release.sh

```diff
@@ -10,6 +10,13 @@ set -o pipefail
 
 ### Functions: General ###
 
+# Run the supplied command in the background so that `errexit` and `pipefail` are honored.
+# Errors will be counted instead of exiting immediately.
+function record_errors() {
+    "$@" &
+    wait $! || errors=$((errors+1))
```

Comment:
Why not

```suggestion
    "$@" || errors=$((errors+1))
```

Reply:
This is exactly the behavior causing the problem...

```bash
bash -c 'set -e; foo() { doesnt_exist; echo STILL HERE $?; }; foo || echo Shouldve failed'
```

Whereas this works:
```bash
bash -c 'set -e; foo() { doesnt_exist; echo STILL HERE $?; }; foo& wait $! || echo FAILED with $?'
```

Reply:
Ah yes, with functions.